certified-connectors/BttnV2/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "PowerAutomate",
+          "scopes": [
+            "powerautomate"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://sso.bt.tn/auth/realms/bttn/protocol/openid-connect/auth"
+            },
+            "tokenUrl": {
+              "value": "https://sso.bt.tn/auth/realms/bttn/protocol/openid-connect/token"
+            },
+            "refreshUrl": {
+              "value": "https://sso.bt.tn/auth/realms/bttn/protocol/openid-connect/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#c51a1b",
+    "capabilities": [],
+    "publisher": "Wessel Bruijnes"
```

Comment:
@WesselBruijnes The publisher name can not be an individual name.  if you want to get this connector certified with an individual name, you need to move your connector to "independent-publisher connectors" folder.